src/components/component-playground.jsx

```diff
@@ -180,13 +181,53 @@ module.exports = React.createClass({
     </ul>;
   },
 
-  _renderContentFrame: function() {
-    return <div ref="contentFrame" className={this._getContentFrameClasses()}>
+  _renderPreview: function() {
+    return (
       <div ref="previewContainer" className={this._getPreviewClasses()}>
         {this.loadChild('preview')}
       </div>
-      {this.props.editor ? this._renderFixtureEditor() : null}
-    </div>
+    );
+  },
+
+  _renderContentFrame: function() {
+    var split = this.state.orientation == 'landscape' ?
+      'vertical' : 'horizontal';
+    var classes = {
+      [style[split]] : true,
+      [style['split-pane']] : true
+    };
+    var resizerClasses = {
+      [style[split]] : true,
+      [style['resizer']] : true
+    }
+
+    classes = classNames(classes);
```

Comment:
For brevity you can write it like this:

```js
var classes = classNames(style[split], style['split-pane']);
```

[classNames](https://github.com/JedWatson/classnames) will ignore undefined values and can accept multiple params instead of an object.